psychopy/app/pavlovia_ui/project.py

```diff
@@ -434,7 +434,7 @@ def syncProject(parent, project=None, closeFrameWhenDone=False):
         # project didn't exist at Pavlovia (deleted?)
         recreatorDlg = ProjectRecreator(parent=parent, project=project)
         ok = recreatorDlg.ShowModal()
-        if ok > 0:
+        if ok is not None:
```

Comment:
Looking at the code for `ProjectRecreator.ShowModal` we shouldn't ever be seeing `None`. We should be returning 1 for "OK" and -1 for "Cancel". I think this change to the code will cause cancel not to cancel anymore

Reply:
Although if there is a `None` case coming out somehow, we should then use `if ok is not None and ok>0:`

Reply:
Ah, yes it returns None because when OK is clicked for choice 3 ("Do nothing and abort the sync"), there is no conditional for choice 3, so it returns None. Although, there is no "cancel" button either. I will push up the change to include your fix, but also need to add a cancel button then?

Reply:
Or let's return -1 from that option (as if it is the cancel button?). We could add the cancel button too I guess

Reply:
Yes, perhaps a cancel button is faster - user can cancel without having to read options and select radio - however just added the return -1. Will look at adding a cancel now.